minimax.py

```diff
@@ -289,10 +290,27 @@ def create_players(player_csv):
     players = [mock_player]
     return players
 
-def update_players_csv(player_lookup):
-    # TODO @Nashawn: update player.csv 
-    # player_lookup holds {lowercased_name, player_object}
-    pass
+# This method takes in the player object and updates the CSV with the latest data from the object at the time of being called.
+
+def update_players_csv(player):
+    
+    newPlayerData = [player.name, player.optimal_moves , player.total_moves, player.optimal_percent]
+
+    with open('players.csv', 'r') as readFile:
```

Comment:
Can you move player.csv to CSVFolder. We are placing all of our CSV files there.